Corona-Warn-App/src/main/java/de/rki/coronawarnapp/service/submission/SubmissionConstants.kt

```diff
@@ -49,6 +49,8 @@ object SubmissionConstants {
 
     const val minKeyCountForSubmission = 14
     const val fakeKeySize = (1 * 16 /* key data*/) + (3 * 4 /* 3x int32*/)
+    const val qrCodeRegex = "^(http:\\/\\/www\\.|https:\\/\\/www\\.|http:\\/\\/|https:\\/\\/)" +
```

Comment:
Please make constants `ALL_CAPS`.

Reply:
This should be checking for https only and for `localhost`. Please also add corresponding unit tests for localhost and other host names.

Reply:
@mlenkeit  the relevant regex has been added

I have included all scenarios in test cases but you're welcome to test yourself [here](https://regexr.com/)

Regex string : ^((^https:\/{2}localhost)(\/\?)[A-Fa-f0-9]{6}[-][A-Fa-f0-9]{8}[-][A-Fa-f0-9]{4}[-][A-Fa-f0-9]{4}[-][A-Fa-f0-9]{4}[-][A-Fa-f0-9]{12})$

Test String: https://localhost/?3D6D08-3567F3F2-4DCF-43A3-8737-4CD1F87D6FDA